ingest-app/src/cmr/ingest/system.clj

```diff
@@ -97,6 +98,10 @@
                                 (af/refresh-acl-cache-job "ingest-acl-cache-refresh")
                                 jvm-info/log-jvm-statistics-job))
               :caches {acl/token-imp-cache-key (acl/create-token-imp-cache)
+                       ;; Caches a map of tokens to the security identifiers
+                       context-augmenter/token-sid-cache-name (context-augmenter/create-token-sid-cache)
```

Comment:
I don't think we need the SIDs in ingest. 

I agree caching the user-id makes sense we are saving it with variable and service concepts, but I think the SIDs caching in search is because we pass the groups a user belongs to as part of the Elasticsearch query. We don't do that in ingest.